init.lua

```diff
@@ -2,6 +2,8 @@
 -- Based on node_ownership
 -- License: LGPLv2+
 
+factions_avail = minetest.global_exists("factions")
```

Comment:
Mods should only define their own table (namespace) in the global environment. `areas.factions_available` would be better